vibetype/ViewController.swift

```diff
@@ -228,21 +232,55 @@ extension UIColor {
 }
 
 extension ViewController: WKScriptMessageHandler {
-  func userContentController(_ userContentController: WKUserContentController, didReceive message: WKScriptMessage) {
-        if message.name == "print" {
+    func returnTrackingPermissionResult(isAuthorized: Bool) {
+        let result = isAuthorized ? "authorized" : "denied"
+        dispatchEventToWebView(name: "tracking-permission-request", data: result)
```

Comment:
How would we listen for this in the `maevsi/vibetype`? Using the service worker?